Stories/Readme.md

```diff
@@ -0,0 +1,52 @@
+Sam wanted to take different specimens home for his terrarium and to show their robot friends, but these fragile life creatures wouldn't make it through the wormholes intact. Sam decided to examine and study how they function so they could build his own self-replicating creatures on his home planet.
+Can you help Sam to decipher the DNA?
+
+
+
+### Story 2: Beer lover Anna wants to investigate craft beers 
+
+- [Google Slides](https://docs.google.com/presentation/d/1FeZkuDztakgAw-oHA6b-mFGYFs3GdNnO3eRAybJGHDU/edit#slide=id.gcf3936e2dc_0_1)
+
+Meet Anna! She is an architect who plays volleyball, likes to go hiking, loves reading and drinking beer. One day she came across an article discussing the beer microbiome.
+This got her wondering what is a beer microbiome? There are collections of small living creatures. Therefore she starts researching. She finds out that these small creatures are called bacteria and they are everywhere. In our gut, in the soil, even on vending machines. Some of these bacteria are actually very good for us. And some others can make us very ill. Bacteria come in different shapes and sizes but they have the same components. One crucial component is the DNA, the blueprint of life. The DNA encodes the shape and size and many other things unique for a bacterial species. Because of the encoding information the DNA can be used to identify what kind of bacteria the DNA is from. Therefore within a sample form soil, our gut or beer one can specify what kind of species are inside the sample. Anna would like to learn more about this kind of analysis. Can you help Anna to find out more?
+
+
+
+### Story 3: wine makers: Jeff wants to decode an old family wine recipe
+
+- [Google Slides](https://docs.google.com/presentation/d/1A8gAgNDjmQ_DfCuIUIkghRv82mlaVCD_ngB6VQ3CfaA/edit#slide=id.p)
+
+Meet Jeff, he is a Winemaker in the third generation. He and his family live on a beautiful vineyard. His favorite wine is an old family recipe. Since they have many bottles left he never had to produce the wine. Unfortunately, one day there was a fire on the farmhouse and the old family recipe got burned. But there are still some bottles of the wine left. Jeff thought about the ingredients of the wine.  He remembered a lot! But especially, the yeast strains used for fermentation he could not remember! He found out that one can sequence the DNA in wine and find out what yeast strains are inside the wine. Now Jeff wants to learn about microbiome data analysis. Can you help him?
+
+- Jeff could also be a beer brewer
+
+### Story 4: crime scene
```

Comment:
here this is more related to human DNA analysis and identify each person.

Reply:
Yes, I added two subtitles to clarify that only the first 3 stories will help us with the microbiome. I think we should also start here. For now, maybe we can leave the other stories if we would like to extend the game at one point?